src/store/reducers/food.js

```diff
@@ -0,0 +1,19 @@
+import { ADD_FOOD, DELETE_FOOD } from '../actions/food';
+
+let initState = {
+  foods: [],
+};
+
+const foodReducer = (state = initState, action) => {
+  switch (action.type) {
+    case ADD_FOOD:
+      return { ...state, foods: state.foods.push(action.payload) };
```

Comment:
I have tried this code, and you have an error on this line. You cannot mutate state like this. My suggestion is
 `foods: [...state.foods, action.payload]`